distributed/protocol/tests/test_numpy.py

```diff
@@ -192,6 +174,7 @@ def test_dumps_large_blosc(c, s, a, b):
 @pytest.mark.skipif(sys.version_info[0] < 3,
                     reason='numpy doesnt use memoryviews')
 def test_compression_takes_advantage_of_itemsize():
+    pytest.importorskip('lz4')
```

Comment:
w/o blosc or lz4 installed:
```
$ py.test distributed/protocol/tests/test_numpy.py
52 passed, 4 skipped, 15 warnings in 0.53 seconds
```
with blosc installed, but not lz4, before:
```
1 failed, 52 passed, 3 skipped, 15 warnings in 0.94 seconds
```
with this change:
```
52 passed, 4 skipped, 15 warnings in 0.60 seconds
```